via/services/google_drive.py

```diff
@@ -4,10 +4,10 @@
 from json import JSONDecodeError
 from typing import ByteString, Iterator
 
-from google.oauth2.service_account import Credentials
-from googleapiclient.discovery import build
+import google.oauth2.service_account
+import googleapiclient.discovery
+import googleapiclient.http
```

Comment:
If you don't fully qualify the imports then patching fails with "no such attribute blah on google" etc.